main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/kinvolk/mountinfo-graph/bindata"
+	"github.com/kinvolk/mountinfo-graph/migraph"
+	"html/template"
+	"log"
+	"net/http"
+)
+
+func generateFromTemplate(htmlTemplate string, w http.ResponseWriter, body string) {
```

Comment:
My idea was to rather pass the result of either `bindata.Index()` or `bindata.Show()` to the `htmlTemplate` parameter, so the map in the function would not be necessary. But now I can see that we also need a name for the template. Question is whether this name is important for us now. If it is not, you could just hardcode some string there like `t, err := template.New("not important").Parse(htmlTemplate)`.

If the name is important, then you could modify the `bindata` functions to return two strings - a name and data and then forward those to the `generateFromTemplate` function.